Documentation/DataSources/DeprecatedWorkbookRetrievalTool.md

```diff
@@ -0,0 +1,352 @@
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "queryType": 12
+          }
+        ],
+        "style": "pills",
+        "queryType": 12
+      },
+      "conditionalVisibility": {
+        "parameterName": "debug",
+        "comparison": "isNotEqualTo"
+      },
+      "name": "turn response into param"
+    },
+    {
+      "type": 11,
+      "content": {
+        "version": "LinkItem/1.0",
+        "style": "list",
+        "links": [
+          {
+            "id": "fc93ee9e-d5b2-41de-b74a-1fb62f0df49e",
+            "linkTarget": "OpenBlade",
+            "linkLabel": "Open Content as Workbook",
+            "style": "primary",
+            "bladeOpenContext": {
+              "bladeName": "UsageNotebookBlade",
+              "extensionName": "AppInsightsExtension",
+              "bladeParameters": [
+                {
+                  "name": "ComponentId",
+                  "source": "parameter",
+                  "value": "resource"
+                },
+                {
+                  "name": "NewNotebookData",
+                  "source": "parameter",
+                  "value": "config"
+                }
+              ]
+            }
+          }
+        ]
+      },
+      "conditionalVisibility": {
+        "parameterName": "config",
+        "comparison": "isNotEqualTo"
+      },
+      "name": "links - 4"
+    }
+  ],
+  "fallbackResourceIds": [
+    "/subscriptions/8980832b-9589-4ac2-b322-a6ae6a97f02b/resourceGroups/tolee-test/providers/Microsoft.OperationalInsights/workspaces/tolee-test-ws"
+  ],
+  "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/workbook.json"
+}
+
+4. Click Apply at the top right
+5. Select the subscription and resource group of the workbook you'd like to retrieve the workbook from
+6. The grid at the bottom of this workbook will list all the private workbooks in the selected subscription / resource group above.
+7. Click on one of the workbooks in the grid. Your workbook should look something like this:
+    > ![advanced editor](../Images/DeprecatedWb_RetrievalTool_PrivateWbConversion.png)
+8. Click the button at the bottom of the workbook labeled "Open Content as Workbook"
+9. A new workbook will open with the exact content of the old private workbook that you selected. You can now treat this like any new workbook.
+
+## Favorites Based (Legacy) Workbook Retrieval
+1. Navigate to your Application Insights Resource > Workbooks gallery
+2. Open up an new or empty workbook
+3. Click Edit in the toolbar and navigate to the advanced editor
+    ![advanced editor](../Images/DeprecatedWb_RetrievalTool_AdvancedEditor.png)
+4. Copy the following workbook json and paste it into your open advanced editor
+>{
```

Comment:
same formatting for code.  OR these should be links to `.workbook` files  that will open in github instead of being inline text?